### PR TITLE
Add minikube on 0.35.0

### PR DESCRIPTION
**Pull request by Magicloud**
Minikube is kind of useful on one host usercase.

Migrated from Archlinux AUR. Tested in Alpinelinux v3.9 x86_64.

### CODE REVIEW

**Comment by Ikke** [comment]
New aports should start in testing, not community.

**Comment by ncopa** [comment]
The commit message needs to be better. The [github contributing guidelines](https://github.com/alpinelinux/aports/blob/master/.github/CONTRIBUTING.md) says:

> Begin the commit message with a single short line summarizing the changes, followed by a blank line and then a more thorough description. Examples of correctly formated summary lines are:
> - When adding a new aport: testing/apkname: new aport

https://github.com/alpinelinux/aports/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request-pr

File: community/minikube/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+pkgname=minikube
+pkgver=0.35.0
+pkgrel=0
+pkgdesc="Minikube is a tool that makes it easy to run Kubernetes locally"
+url="https://github.com/kubernetes/minikube"
+arch="all"
+license="Apache"
+makedepends="make go git"
+source="$pkgname-$pkgver.tar.gz::https://github.com/kubernetes/minikube/archive/v$pkgver.tar.gz"
+options="!check"
```
Comment (TBK) [inline community/minikube/APKBUILD]: Please add a comment on why check is disabled.

**Comment by Magicloud** [comment]
Close this one since it is on wrong branch.

**Pull request closed without merging**